dashboard/test/controllers/api/v1/teacher_feedbacks_controller_test.rb

```diff
@@ -0,0 +1,155 @@
+require 'test_helper'
+
+class Api::V1::TeacherFeedbacksControllerTest < ActionDispatch::IntegrationTest
+  API = '/api/v1/teacher_feedbacks'
+  COMMENT1 = 'Comment Alpha'
+  COMMENT2 = 'Comment Beta'
+  COMMENT3 = 'Comment Gamma'
+
+  setup do
+    #create student, teacher, and level and register student in teacher's section
+    @teacher = create :teacher
+    @student = create :student
+    @section = create :section, user: @teacher
+    @section.add_student(@student)
+    @level = create :level
+  end
```

Comment:
This can be done once and shared across test cases by adding the line:
```ruby
  self.use_transactional_test_case = true
```
and changing `setup` to `setup_all`. 

Note - this is something Will [wrote](https://github.com/code-dot-org/code-dot-org/pull/13569) in our code base, not part of Rails, but it has been [submitted upstream](https://github.com/rails/rails/pull/28178).

Reply:
Oh this is awesome! I was looking for something like this in Rails!